util.js

```diff
@@ -2,6 +2,7 @@
 const fs = require('fs');
 const path = require('path');
 const pkg = require('./package');
+const esmRequire = require('esm')(module);
```

Comment:
To be [in sync with AVA itself](https://github.com/avajs/ava/blob/b6a95b14099df88d33b12b998f41fa9bb2bbc60d/lib/load-config.js#L17:L20), you need to set `{cjs: false, mode: 'all'}` options.